### PR TITLE
fix(DB/gossip_menu_option): Give me a bomber!

### PR DESCRIPTION
**Pull request by LukasVolgger**
## CHANGES PROPOSED:
- Remove duplicated gossip menu option (TC also doesn't have it or deleted it)

Before
2020-12-01 11_04_28-World of Warcraft

After
2020-12-01 11_11_14-World of Warcraft

## ISSUES ADDRESSED:
- None

## TESTS PERFORMED:
- Applied SQL
- Tested ingame (See Screenshots)

## HOW TO TEST THE CHANGES:

- .q rem 13373 & .q a 13373
- .go c 1977169
- Talk to the NPC, he should have only one gossip menu option
- Complete quest 13373 and be sure that nothing is broken
## KNOWN ISSUES AND TODO LIST:

- [ ]
- [ ] 

## Target branch(es):
- [x] Master

 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1606850752274331900.sql [hunk 1]
```diff
@@ -0,0 +1,3 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1606850752274331900');
+
+DELETE FROM `gossip_menu_option` WHERE  `MenuID`=10120 AND `OptionID`=1;
```
Comment (Si1ker) [inline data/sql/updates/pending_db_world/rev_1606850752274331900.sql:3]:
```suggestion
DELETE FROM `gossip_menu_option` WHERE  `MenuID`=10120 AND `OptionID`=1;

```

**Review by Si1ker — CHANGES_REQUESTED** [review]

**Review by Si1ker — APPROVED** [review]

**Comment by Si1ker** [comment]
@LukasVolgger 

core-build / ubuntu-20.04-clang-with-modules (pull_request) Failing after 4m — ubuntu-20.04-clang-with-modules

(and many others...)

2020-12-01 20:50:39 addToGossipMenuItems: GossipMenuId 10120 Item 1 not found
2020-12-01 20:50:39 Not handled grouped condition, SourceGroup 10120
2020-12-01 20:50:39 addToGossipMenuItems: GossipMenuId 10120 Item 1 not found
2020-12-01 20:50:39 Not handled grouped condition, SourceGroup 10120

**Review by FrancescoBorzi — APPROVED** [review]

File: data/sql/updates/pending_db_world/rev_1606850752274331900.sql (hunk 1 above)
Comment (FrancescoBorzi) [inline data/sql/updates/pending_db_world/rev_1606850752274331900.sql:3]: this change is causing startup errors:

```
2020-12-01 20:50:17 addToGossipMenuItems: GossipMenuId 10120 Item 1 not found
2020-12-01 20:50:17 Not handled grouped condition, SourceGroup 10120
2020-12-01 20:50:17 addToGossipMenuItems: GossipMenuId 10120 Item 1 not found
2020-12-01 20:50:17 Not handled grouped condition, SourceGroup 10120
```

so the build fails

**Review by FrancescoBorzi — CHANGES_REQUESTED** [review]

**Comment by LukasVolgger** [comment]
Updated conditions - Source TC DB. Please test!

**Comment by pklloveyou** [comment]
Tested

**Comment by FrancescoBorzi** [comment]
thanks @LukasVolgger and @pklloveyou !

**Pull request merged**